src/cow-react/modules/application/containers/App/RoutesApp.tsx

```diff
@@ -41,43 +45,154 @@ function createRedirectExternal(url: string) {
 export function RoutesApp() {
   return (
     <Suspense fallback={<Loader />}>
-      <Switch>
-        {/* Optimistic routes */}
-        <Route strict path={Routes.ACCOUNT} component={Account} />
-        {/* Lazy routes */}
-        <Suspense fallback={Loading}>
-          <Switch>
-            <Redirect from="/claim" to={Routes.ACCOUNT} />
-            <Redirect from="/profile" to={Routes.ACCOUNT} />
-            {/*Redirect from the old URL format to a new one*/}
-            <Route exact strict path="/swap" component={NewSwapPageRedirect} />
-            <Route exact path={Routes.SWAP} component={NewSwapPage} />
-            <Route exact path={Routes.LIMIT_ORDER} component={LimitOrders} />
-            <Route exact strict path={Routes.SEND} component={RedirectPathToSwapOnly} />
-            <Route exact strict path={Routes.ABOUT} component={About} />
+      <Routes>
+        {/*Account*/}
+        <Route path={RoutesEnum.ACCOUNT} element={<Account />}>
+          <Route path={RoutesEnum.ACCOUNT} element={<AccountOverview />} />
+          <Route path={RoutesEnum.ACCOUNT_TOKENS} element={<AccountTokensOverview />} />
+          <Route path="*" element={<AccountNotFound />} />
+        </Route>
+        <Route path="claim" element={<Navigate to={RoutesEnum.ACCOUNT} />} />
+        <Route path="profile" element={<Navigate to={RoutesEnum.ACCOUNT} />} />
+
+        {/*Swap*/}
+        <Route path={RoutesEnum.SWAP} element={<NewSwapPage />} />
+        <Route path={RoutesEnum.SEND} element={<RedirectPathToSwapOnly />} />
+
+        {/*Limit orders*/}
+        <Route
```

Comment:
There's a bit of repetition here, should we just do a loop?

Reply:
<img width="772" alt="image" src="https://user-images.githubusercontent.com/7122625/213412573-7bfd1d43-7198-47c0-8d32-82465e686913.png">

Thanks! Looks fancy :)